models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql

```diff
@@ -0,0 +1,541 @@
+{#
+Dashboards Store - Helping students, one dashboard at a time.
+Copyright (C) 2023  Sciance Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#}
+with
+    -- Prend les résultats au volet de la matière Français 5. Le volet est un critère
+    -- à l'obtention pour ce cours.
+    _volet as (
```

Comment:
Ta 1ere requete ici devrait etre les eleves actuellement inscrits en secondaire 5 (stg_perimetre_eleve_diplomation_des = {{ get_current_year() }}). Je vais appeler cette table perim
Ensuite tu dois aller recuperer les resultats des matieres d'interets (mat IN matiere_evalue) pour les fiches qui appartiennent à ton perimetre d'analyse (fiche in perim). Je ne sais pas si le INNER JOIN est plus efficace que le fiche IN (SELECT DISTINCT fiche FROM perim)

Cette maniere de proceder rendrait le code plus clair non?

Reply:
Selon Mohamed c'est mieux de faire des INNER plutot que de filtrer ds la clause WHERE dc on laisse comme ca ;)